BE/src/Validation/UserValidationRules/userUpdate.ts

```diff
@@ -1,12 +1,13 @@
 import { body } from "express-validator";
 
 export const updateUserValidationRules = [
-    body("permissions")
-        .custom((value) => {
-            if (!Array.isArray(value)) {
-                throw new Error("Permissions must be an array.");
-            }
-            return true;
```

Comment:
We need to ensure that usernames are < 16 characters long, unless you wanna update that length in the db settings.

They must also contain no spaces. Here are the accepted characters.![image](https://github.com/user-attachments/assets/1317b715-d6d6-4896-a134-75d0230f90d2)

Reply:
I updated the length and the user is notified of failures when they occur.